static/includes/Reference/TasksReplicationAddWizardFields.md.part

```diff
@@ -1,27 +1,51 @@
+| SSH Transfer Security | Data transfer security. The connection is authenticated with SSH. Data can be encrypted during transfer for security or left unencrypted to maximize transfer speed. Encryption is recommended, but can be disabled for increased speed on secure networks. |
+
+**Destination Location: On this System**
+
+| Name | Desciption |
+|------|------|
+| Destination | Define the path to a system location that will store replicated snapshots. Click the folder to see all locations on the destination system or click in the field to manually type a location path (Example: pool1/dataset1). Selecting a location defines the full path to that location as the destination. Appending a name to the path will create new zvol at that location. For example, selecting pool1/dataset1 will store snapshots in dataset1, but clicking the path and typing /zvol1 after dataset1 will create zvol1 for snapshot storage. |
+| Encryption | Set to use encryption when replicating data. Additional encryption options will appear. |
+
 **Destination Location: On a Different System**
 
-| | |
-|-|-|
+| Name | Desciption |
+|------|------|
 | SSH Connections | Select a saved remote system SSH connection or choose Create New to create a new SSH connection. |
 | Destination | Define the path to a system location that will store replicated snapshots. Click the folder to see all locations on the destination system or click in the field to manually type a location path (Example: pool1/dataset1). Selecting a location defines the full path to that location as the destination. Appending a name to the path will create new zvol at that location. For example, selecting pool1/dataset1 will store snapshots in dataset1, but clicking the path and typing /zvol1 after dataset1 will create zvol1 for snapshot storage. |
 | Encryption | Set to use encryption when replicating data. Additional encryption options will appear. |
-| SSH Transfer Security | Data transfer security. The connection is authenticated with SSH. Data can be encrypted during transfer for security or left unencrypted to maximize transfer speed. Encryption is recommended, but can be disabled for increased speed on secure networks. |
+
+### When
+
+| Name | Desciption |
+|------|------|
+| Replication Schedule | Text |
+| Destination Snapshot Lifetime | When replicated snapshots are deleted from the destination system. *Same as Source*: use the configured Snapshot Lifetime value from the source dataset periodic snapshot task. *Never Delete*: never delete snapshots from the destination system. *Custom*: set a how long a snapshot remains on the destination system. Enter a number and choose a measure of time from the drop-down. |
+| Schedule | Set specific times to snapshot the Source Datasets and replicate the snapshots to the Destination Dataset. Select a preset schedule or choose Custom to use the advanced scheduler. |
```

Comment:
there are two ways to address the capitals in this sentence, if the UI element make it clear you are talking about the value in these fields, but the way this reads it is a general mention of these so no capitals. suggestion 1:
```suggestion
| Schedule | Set specific times to snapshot what you specified in **Source Datasets** and replicate the snapshots to the location in **Destination Dataset**. Select a preset schedule or choose **Custom** to use the advanced scheduler. |
```
or no capitals
```suggestion
| Schedule | Set specific times to snapshot the source datasets and replicate the snapshots to the destination dataset. Select a preset schedule or choose Custom to use the advanced scheduler. |
```